utils/switcher/switcher.js

```diff
@@ -255,6 +258,23 @@ const fuzzGoalBufferLength = function() {
   return result;
 };
 
+const fuzzBufferLowWaterLine = function() {
+  let result = [];
+
+  let BLWL = Math.max(1, Number($('#buffer-low-water-line').value));
+  let BLWLStep = Math.max(1, Number($('#buffer-low-water-line-step').value));
+  let BLWLMax = Math.max(BLWL, Number($('#buffer-low-water-line-max').value));
+
+  result.push(BLWL);
```

Comment:
Can change this and the below to either a do while loop to avoid this being outside the loop, or a for loop:

```
for (; BLWL <= BLWLMax; BLWL += BLWLStep) {
  result.push(BLWL);
}
```